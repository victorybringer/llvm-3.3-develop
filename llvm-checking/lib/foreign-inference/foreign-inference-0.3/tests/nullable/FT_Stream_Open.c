#include <sys/types.h>
#include <fcntl.h>

void FT_Stream_Open(const char *filepathname) {
  open(filepathname, O_RDONLY);
}
