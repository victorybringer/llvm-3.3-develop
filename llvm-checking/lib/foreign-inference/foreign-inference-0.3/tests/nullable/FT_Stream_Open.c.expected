fromList [ ("@FT_Stream_Open", fromList ["%filepathname"]) ]
